tests/js/web/PulsarUIComponentTest.js

```diff
@@ -3,10 +3,13 @@
 'use strict'
 
 var $ = require('jquery'),
+    dropdown = require('../../../js/libs/dropdown'),
     history = require('../../../libs/history.js/scripts/bundled/html5/jquery.history'),
     tab = require('../../../js/libs/tab'),
     PulsarUIComponent = require('../../../js/PulsarUIComponent');
 
+    $.fx.off = !$.fx.off;
```

Comment:
Do we intentionally want to flip this boolean? Or do we always want it to be `true` for the tests?

Reply:
I was trying to avoid having to setTimeout just to get some tests to pass, not sure this did anything tbh

Reply:
It should disable the animations, whether this makes them behave synchronously (I'm not sure, probably not), my point is if we do want to do this, we should probably be explicit and set ` $.fx.off = true`